src/components/ui/ComboBox.tsx

```diff
@@ -6,29 +6,43 @@ function classNames (...classes): string {
   return classes.filter(Boolean).join(' ')
 }
 
-interface ValueObj{
+export interface ValueObj{
   id: number
   name: string
+  disabled?: boolean
 }
 
-interface ComboBoxProps{
+interface ComboBoxSingleProps {
   options: ValueObj[]
   value: ValueObj
   onChange: (value: ValueObj) => void
   label: string
+  selectClassName?: string
 }
-export default function ComboBox ({ options, value, onChange, label }: ComboBoxProps): JSX.Element {
+interface ComboBoxMultiProps {
+  options: ValueObj[]
+  value: ValueObj[]
+  onChange: (value: ValueObj[]) => void
+  label: string
+  multiple: true
+  selectClassName?: string
+}
+type ComboBoxProps = ComboBoxSingleProps | ComboBoxMultiProps
+
+export default function ComboBox (props: ComboBoxProps): JSX.Element {
+  const { options, value, onChange, label, selectClassName = '' } = props
   return (
-    <Listbox value={value} onChange={onChange}>
+    <Listbox value={value} onChange={onChange} multiple={'multiple' in props ? props.multiple : false}>
       {({ open }) => (
         <>
-          <Listbox.Label className='block text-sm font-medium text-gray-700 mt-2'>{label}</Listbox.Label>
+          {label !== '' && <Listbox.Label className='block text-sm font-medium text-gray-700 mt-2'>{label}</Listbox.Label>}
```

Comment:
So that `MultiSelect` can insert its own label with formatting consistent across all form inputs.